.github/workflows/docker-test-devcontainer.yml

```diff
@@ -0,0 +1,28 @@
+name: Docker test build (Devcontainer)
+
+on:
+  push:
+    branches: 
+    - '*'
+
+  pull_request:
+    branches: 
+    - '*'
+
+jobs:
+  test-build-latest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v1
+        with:
+          file: ./.devcontainer/Dockerfile
```

Comment:
Looking through [one Docker test build](https://github.com/Yuuki-Discord/Yuuki-Bot/runs/1163457509) underneath the "Build and push Docker image" step, I notice that it warns regarding the usage of `file`:
> Warning: Unexpected input(s) 'file', valid inputs are ['entryPoint', 'args', 'username', 'password', 'registry', 'repository', 'tags', 'tag_with_ref', 'tag_with_sha', 'path', 'dockerfile', 'target', 'always_pull', 'build_args', 'cache_froms', 'labels', 'add_git_labels', 'push']

I think that this skips over using `./.devcontainer/Dockerfile` and uses `./Dockerfile` from the root of this repository as it finalises its build with `CMD ["./run_linux.sh"]`, whereas the development one is `CMD ["sleep", "infinity"]`. Could you please check and see if that's what's happening?

Additionally, is it worth investigating the v2 beta of this action as it warns?

Reply:
Excellent point, thank you for spotting this!

You are correct, it was building the old one since the `file:` was ignored.
I've updated the devcontainer test build to v2 and it appears to work now, was a pain to debug since I had no way to test locally and it kept coming up with new deprecations to replace or remove.

Thanks again!